proto-google-common-protos/src/main/java/com/google/api/MetricDescriptor.java

```diff
@@ -2267,11 +2267,11 @@ public int getValueTypeValue() {
    * The units in which the metric value is reported. It is only applicable
    * if the `value_type` is `INT64`, `DOUBLE`, or `DISTRIBUTION`. The `unit`
    * defines the representation of the stored metric values.
-   * Different systems may scale the values to be more easily displayed (so a
-   * value of `0.02KBy` _might_ be displayed as `20By`, and a value of
-   * `3523KBy` _might_ be displayed as `3.5MBy`). However, if the `unit` is
-   * `KBy`, then the value of the metric is always in thousands of bytes, no
-   * matter how it may be displayed..
+   * Different systems might scale the values to be more easily displayed (so a
```

Comment:
why might instead of may? Do you think scaling is more or less likely to be true?